### PR TITLE
Rethemes the ash walker nest to match the Necropolis

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
tweak: Rethemes the ash walker nest to match the Necropolis.
/:cl:

Added: Wood, tower cap seeds, a water tank, a water bottle, a recharging experimental welder.
Removed: Like fifty flashlights in favor of mining lanterns.
traditional carp port ten 2017-06-27 184913
traditional carp port ten 2017-06-27 184916

### CODE REVIEW

**Comment by FantasticFwoosh** [comment]
Lava surrounding the tendril is just aesthetic right?

**Comment by GunHog** [comment]
I assume it is, as the eggs are not burning!

**Comment by ChangelingRain** [comment]
It's not aesthetic but the tiles prevent the eggs from burning.

**Comment by GunHog** [comment]
Is it safe to walk on it, then? If not, a lot of Ashwalkers will become **_delicious._**

**Comment by ChangelingRain** [comment]
It is safe. It's technically aesthetic, but the tiles are real.

**Comment by ShizCalev** [comment]
tgm?

**Comment by ChangelingRain** [comment]
I used it. It didn't work. Please debug.

**Comment by ShizCalev** [comment]
worked fine for me.

switch to master, do a prepare map, then switch back and try again

**Comment by FantasticFwoosh** [comment]
@ChangelingRain i mean there is no visible aesthetic tile directly underneath the tendril.

**Comment by ChangelingRain** [comment]
You can't get to that tile normally. The tendril is unaffected by lava, because it is a tendril.

**Comment by FantasticFwoosh** [comment]
Yes but when it is destroyed it leaves a singularity core, which will burn in the lava because its the loot drop.

**Comment by ChangelingRain** [comment]
Did you read the code?

**Comment by FantasticFwoosh** [comment]
Not sure where that particular commit would be given this literally leads on from your previous PR, but initially no for a handful of reasons.

**Comment by ChangelingRain** [comment]
If you read the code, you'd know that the core is not on the middle turf when it dies.

**Comment by FantasticFwoosh** [comment]
Well that's new, not that it'd be a needed change because everyone is going to run into the exposed lava tile and probably burn it. OR burn the eggs after the tendril has been destroyed.

**Comment by Repukan** [comment]
Misleading title

**Comment by Penguaro** [comment]
I like it. It reminds me of Dungeon Keeper.

**Comment by Armhulen** [comment]
you reminded me of dungeon keeper :(

**Comment by Bawhoppen** [comment]
why such symmetry with the nest room???

**Comment by jughu** [comment]
>removed the puddle WEEO WEEO UNDOCUMENTED CHANGE

**Comment by DoctoreMad** [comment]
Err, can you give them some cult metal like runed metal? I'm going to miss the runed golem reward for killing the ash liggers.

**Comment by FantasticFwoosh** [comment]
@DoctoreMad that was not intentional because the new theme is not culty. You'll just have to go and bother the nar-sie ruin

**Comment by Ergovisavi** [comment]
I'm just going to throw this out there but please remove any source of water (like that watertank).

I already kill enough people on station as is with a glass of water, this just gives Ash Lizards easy access to a cheap stun. Yes, water being bizarrely effective stun is the real problem, but the last time I tried to nuke it off the face of the earth it didn't pass.

**Comment by FantasticFwoosh** [comment]
@Ergovisavi a alternative solution is to simply copy paste the code, make the dense modification and then supply a alternative sprite for a non-craftable ashwalker bonfire.

**Comment by Hatfish** [comment]
If you remove the watertank then please be sure to supply seeds that don't require (much) water instead so they aren't completely useless.

**Comment by ChangelingRain** [comment]
I'd rather make basalt not wettable at all rather than totally remove their water, but that's better suited for a different pr.

**Comment by AnturK** [comment]
This ready ?

File: code/modules/mob/living/simple_animal/spawner.dm [hunk 1]
```diff
@@ -30,10 +30,9 @@
 	spawned_mobs = null
 	return ..()
 
-/mob/living/simple_animal/hostile/spawner/Life()
-	..()
-	if(!stat)
```
Comment (AnturK) [inline code/modules/mob/living/simple_animal/spawner.dm]: Are there any spawners that do not self delete ?

**Comment by lzimann** [comment]
@KorPhaeron

**Comment by Repukan** [comment]
It's a funny time when retheme gains the same meaning as both redesign and buff

**Comment by ChangelingRain** [comment]
Yes, I consider this ready.

File: code/modules/mob/living/simple_animal/spawner.dm (hunk 1 above)
Comment (ChangelingRain) [inline code/modules/mob/living/simple_animal/spawner.dm]: It doesn't matter, since stat is checked before `handle_automated_action()` is called

**Comment by KorPhaeron** [comment]
Should remove the water because of what ergo said otherwise this is good

**Comment by KorPhaeron** [comment]
Oh wait I see your other PR

Will merge both together

**Comment by adamkad1** [comment]
if your gonna consider removing water, add shovel plox... also can ashliggers even tan leather now?  didnt you make goliath cloak for them and hermits? also add the bowl back plox, i cant make soup in mushroom bowl

**Pull request merged**